sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/DataFeedLiveTests.cs

```diff
@@ -1445,6 +1462,204 @@ public async Task UpdateSqlServerDataFeedWithCredentialAuthentication(string aut
+                {
+                    RollupType = DataFeedRollupType.RollupNeeded,
+                    AutoRollupMethod = DataFeedAutoRollupMethod.Sum,
+                    RollupIdentificationValue = "dimension"
+                },
+                MissingDataPointFillSettings = new DataFeedMissingDataPointFillSettings(DataFeedMissingDataPointFillType.NoFilling),
+                AccessMode = DataFeedAccessMode.Public,
+                ActionLinkTemplate = "https://fakeurl.com/%datafeed/%metric"
+            };
+
+            await using var disposableDataFeed = await DisposableDataFeed.CreateDataFeedAsync(adminClient, dataFeedToCreate);
+
+            DataFeed dataFeedToUpdate = disposableDataFeed.DataFeed;
+
+            dataFeedToUpdate.IngestionSettings.IngestionStartOffset = null;
+            dataFeedToUpdate.IngestionSettings.IngestionRetryDelay = null;
+            dataFeedToUpdate.IngestionSettings.StopRetryAfter = null;
+            dataFeedToUpdate.IngestionSettings.DataSourceRequestConcurrency = null;
+            dataFeedToUpdate.Schema.TimestampColumn = null;
+            dataFeedToUpdate.Description = null;
+            dataFeedToUpdate.RollupSettings.RollupType = null;
+            dataFeedToUpdate.RollupSettings.AutoRollupMethod = null;
+            dataFeedToUpdate.RollupSettings.RollupIdentificationValue = null;
+            dataFeedToUpdate.MissingDataPointFillSettings = null;
+            dataFeedToUpdate.AccessMode = null;
+            dataFeedToUpdate.ActionLinkTemplate = null;
+
+            DataFeed updatedDataFeed = await adminClient.UpdateDataFeedAsync(dataFeedToUpdate);
+
+            Assert.That(updatedDataFeed.IngestionSettings.IngestionStartOffset, Is.EqualTo(TimeSpan.Zero));
+            Assert.That(updatedDataFeed.IngestionSettings.IngestionRetryDelay, Is.EqualTo(TimeSpan.FromSeconds(-1)));
+            Assert.That(updatedDataFeed.IngestionSettings.StopRetryAfter, Is.EqualTo(TimeSpan.FromSeconds(-1)));
+            Assert.That(updatedDataFeed.IngestionSettings.DataSourceRequestConcurrency, Is.EqualTo(-1));
+            Assert.That(updatedDataFeed.Schema.TimestampColumn, Is.Empty);
+            Assert.That(updatedDataFeed.Description, Is.Empty);
+            Assert.That(updatedDataFeed.RollupSettings.RollupType, Is.EqualTo(DataFeedRollupType.NoRollupNeeded));
+            Assert.That(updatedDataFeed.RollupSettings.AutoRollupMethod, Is.EqualTo(DataFeedAutoRollupMethod.None));
+            Assert.That(updatedDataFeed.RollupSettings.RollupIdentificationValue, Is.Null);
+            Assert.That(updatedDataFeed.MissingDataPointFillSettings.FillType, Is.EqualTo(DataFeedMissingDataPointFillType.SmartFilling));
+            Assert.That(updatedDataFeed.AccessMode, Is.EqualTo(DataFeedAccessMode.Private));
+            Assert.That(updatedDataFeed.ActionLinkTemplate, Is.Empty);
+        }
+
+        [RecordedTest]
+        public async Task UpdateAzureBlobDataFeedAuthenticationWithNullSetsToDefault()
```

Comment:
The following tests handle properties specific to each data feed source. The test above was for common properties only.